src/client/unity.yml

```diff
@@ -0,0 +1,280 @@
+openapi: "3.0.0"
+info:
+  title: Unity API Service
+  version: 0.1.0
+servers:
+  - url: /unity/v2 # to be determined
+paths:
+  "/orgs/{orgID}/invites":
```

Comment:
Should there also be a `get:` method to fetch the list of invites?

Reply:
The list of invites comes back when fetching the user list: you get back both users and invites in that response.